tardis/io/logger/logger.py

```diff
@@ -112,6 +113,9 @@ def logging_state(log_level, tardis_config, specific):
 
         if specific:
             specific = specific
```

Comment:
Instead of an if..else block, why not `if not specific`?

Reply:
I believe this is still relevant

Reply:
Yes, I had the implementation wrong in the first place. Will change it accordingly :)